app/assets/javascripts/fae/navigation/_global_search.js

```diff
@@ -0,0 +1,116 @@
+/* global Fae, FCH */
+
+'use strict';
+
+/**
+ * Fae global search
+ * @namespace globalSearch
+ * @memberof Fae
+ */
+Fae.globalSearch = {
+
+  init: function() {
+    this.setElements();
+    this.utilitySearch();
+    this.searchListener();
+  },
+
+  /**
+   * Caches common elements into globally accessible variables
+   */
+  setElements: function() {
+    this.$wrapper = $('.utility-search-wrapper');
```

Comment:
Prefer to avoid CSS selectors in JS - could you add a `js-` prefixed-ID selector to `.utility-search-wrapper`?